nixos/modules/services/web-servers/nginx/default.nix

```diff
@@ -449,6 +449,24 @@ in
         '';
       };
 
+      defaultHTTPListenPort = mkOption {
+        type = types.port;
+        default = 80;
+        example = 80;
+        description = lib.mdDoc ''
+          If vhosts do not specify listen.port, use these ports for HTTP by default.
+        '';
+      };
+
+      defaultSSLListenPort = mkOption {
+        type = types.port;
+        default = 443;
+        example = 443;
```

Comment:
```suggestion
        example = 8443;
```

Reply:
> How does this interact with other modules like acme?

I checked acme config and it doesn't rely on default listen ports. Probably it will break with webroot setup because some challanges require 80/443/8443. In that case use has to make sure the website is reachable with these ports.

This PR aims to provide a lazy way to setup nginx behind some local load balancer by listening on arbitrary ports. If such load balancer exists, then acme should work fine.

Reply:
My point was that the example should not repeat the default.